Resources/views/Security/base_login.html.twig

```diff
@@ -83,7 +83,7 @@ file that was distributed with this source code.
         </div>
         <div class="col-sm-6">
             {% block sonata_user_registration %}
-                {% render controller("SonataUserBundle:RegistrationFOSUser1:register") %}
+                {% render controller("SonataUserBundle:RegistrationFOSUser2:register") %}
```

Comment:
I think you can trust in some service that is only defined in `2.x`, like [```fos_user.listener.authentication```](https://github.com/FriendsOfSymfony/FOSUserBundle/commit/95d57add6cdd7c535a3471bc750d3ea15e347ecf#diff-a431e3d5d9e6fd8d0548da7680427fb4R8) by instance.